server/report_builder/src/build.rs

```diff
@@ -47,48 +47,58 @@ fn parse_default_query(input: &str) -> anyhow::Result<DefaultQuery> {
 fn extract_sql_entry(
     args: &BuildArgs,
     files: &mut HashMap<String, PathBuf>,
-) -> Result<Option<(String, SQLQuery)>> {
-    match (&args.query_sqlite, &args.query_postgres) {
-        (None, None) => Ok(None),
-        (Some(query_sqlite), Some(query_postgres)) => {
+) -> Result<Vec<SQLQuery>> {
+    let Some(sql_queries) = &args.query_sql else {
+        return Ok(vec![]);
+    };
+    let result: Result<Vec<_>> = sql_queries
+        .iter()
+        .map(|query| {
+            let common_query = format!("{query}.sql");
```

Comment:
If we replace a common query with specific ones for postgres and sqlite is there a risk the common query might be left behind on disk and cause a bug?

Reply:
I guess it's all in a single json record sent from central?

Reply:
Yes.

If there is a common query and a sqlite/postgres file, I could add a warning/error? For example, if the user provided a myquery.sqlite.sql and a myquery.sql file the sqlite file get silently ignored right now...